### PR TITLE
adaption to one of 26 cantons

### PR DESCRIPTION
**Pull request by rebert**
maybe you one of you has a better solution...

[try](https://mf-chsdi3.dev.bgdi.ch/ltrea_hack_ignore_tls_cert_in_request/shorten/8b0824cac1)

### CODE REVIEW

**Comment by rebert** [comment]
context: https://jira.swisstopo.ch/browse/BGDIDIC-1877

File: chsdi/templates/htmlpopup/oerebkataster.mako [hunk 1]
```diff
@@ -15,10 +15,10 @@ def sanitise_url(url):
 def get_xml(path):
     list_egrid = []
     try:
-      response = requests.get(path)
+      response = requests.get(path, verify=False) # verify=False to get it working for grison
```
Comment (ltshb) [inline chsdi/templates/htmlpopup/oerebkataster.mako:18]: I would not set it to False this is a security issue. The issue is due to the Root Certificate used by *.geo.gr.ch (SwissSign AG), it seems that not all browsers know it (at least Firefox 79.0 on ubuntu don't but Chrome 84.0.4147.125 on ubuntu does).
I would suggest to get the Root Certificate from SwissSign and install it on the machine running the mf-chsdi3. See https://requests.readthedocs.io/en/master/user/advanced/#ssl-cert-verification

Comment (rebert) [reply chsdi/templates/htmlpopup/oerebkataster.mako:18]: This sounds like a professional solution

**Comment by ltclm** [comment]
@rebert python2 tests are failing.

**Comment by rebert** [comment]
@p1d1d1 is in contact with canton Grison to get their certificates right

**Pull request closed without merging**